python/semantic_kernel/connectors/ai/open_ai/services/open_ai_text_embedding.py

```diff
@@ -52,26 +52,31 @@ def __init__(
         self._org_id = org_id
         self._log = log if log is not None else NullLogger()
 
-    async def generate_embeddings_async(self, texts: List[str]) -> ndarray:
+    async def generate_embeddings_async(
+        self, texts: List[str], batch_size: int = 0
```

Comment:
default batch size should be 1

```suggestion
        self, texts: List[str], batch_size: int = 1
```

Reply:
Made it Optional[int] = None